src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+
+    get info() {
+      return `name: ${this.name}, `
+            + `chip version: ${this.version}, `
+            + `wheels: ${this.wheels}`;
+    },
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward: function(wayLength = 1) {
```

Comment:
You can use short form of an object method declaration
```suggestion
    goForward(wayLength = 1) {
```